metrics/internal/config/config.go

```diff
@@ -12,6 +12,7 @@ type EnvConfig struct {
 	DatabaseUrl                      string `env:"METRICS_DATABASE_URL,required"`
 	DatabaseMaxConn                  string `env:"METRICS_DATABASE_MAX_CONN"`
 	DatabaseMaxIdleConn              string `env:"METRICS_DATABASE_MAX_IDLE_CONN"`
+	HttpPort                         string `env:"METRICS_HTTP_PORT"`
```

Comment:
Why don't you make it as uint32? And you can set a default value here: `env:"METRICS_HTTP_PORT",envDefault:"3000",notEmpty`